net-misc/asterisk/asterisk-13.29.1.ebuild

```diff
@@ -0,0 +1,322 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit autotools linux-info systemd
+
+MY_P="${PN}-${PV/_/-}"
+
+DESCRIPTION="Asterisk: A Modular Open Source PBX System"
+HOMEPAGE="http://www.asterisk.org/"
+SRC_URI="http://downloads.asterisk.org/pub/telephony/asterisk/releases/${MY_P}.tar.gz
+	http://downloads.uls.co.za/gentoo/asterisk/gentoo-asterisk-patchset-4.08.tar.bz2"
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+
+IUSE_VOICEMAIL_STORAGE="
+	+voicemail_storage_file
+	voicemail_storage_odbc
+	voicemail_storage_imap
+"
+IUSE="${IUSE_VOICEMAIL_STORAGE} alsa bluetooth calendar +caps cluster curl dahdi debug doc freetds gtalk http iconv ilbc xmpp ldap libedit libressl lua mysql newt +samples odbc osplookup oss pjproject portaudio postgres radius selinux snmp span speex srtp static statsd syslog vorbis"
```

Comment:
`xmpp` seems out of place in this list :)
What exaclty does `oss` do, is it still supported?

Reply:
It builds chan_oss, which is still alive.  I've personally never used and don't mind ditching it as "not supported".  It still builds ... which almost comes as a surprise.

xmpp seems out of place but it's not :).  It's used to integrate with gtalk which uses the xmpp protocol, as well as provides for publishing status/availability information to XMPP in general.  Not sure if we have any actual users currently.  I've been threatening to mess with it too and have another project upcoming where it may be useful.

Reply:
I meant in alphabetical order, that you seem to follow everywhere else in this ebuild...